eslint.config

```diff
@@ -0,0 +1,12 @@
+{
+  "parserOptions": { "ecmaVersion": 6 },
+  "extends": "airbnb-base",
+    rules: {
+    'no-shadow': 0,
+    'no-param-reassign': 0,
+    'eol-last': 0,
+  },
+  "globals": {
+    "document": false,
+  }
+}
```

Comment:
**[ OPTIONAL ]** This would be normally marked as required, but for future refference lets stick to the linter configuration provided, which is this one 
![image](https://user-images.githubusercontent.com/46115725/71824161-7a970c80-305e-11ea-9355-916b367e3950.png)

You have a couple of things that don't match the provided config, but they are not code-breaking or letting you get away with anything big so this is just an optional change.

Reply:
Fixed by using the recommended Eslint configuration.